cypress/e2e/tests/login.test.ts

```diff
@@ -22,6 +22,7 @@ describe("Log In", () => {
     it("Login to Pathfinder", { tags: "@tier1" }, () => {
         // Login
         login();
+        cy.visit("/");
```

Comment:
question: is there a reason you're adding this to each `beforeEach` hook instead of adding it to the `login` function or the global `beforeach` ? I just want to understand the reasoning

Reply:
The login function returns a session, but the code that happens in the session is not the context for the test. 

The visit could be added to `beforeEach,` but I think it is ok for the tests to call this as it is clearer the state at which the test is working on. With that said, I didn't go through and fix all the tests, I just tried to fix up `@ci,@tier0@tier1` so that they can run in a reasonable time, and we can add them back to the global CI to get better test coverage.